lab-said/README.md

```diff
@@ -0,0 +1,26 @@
+## Configuration  
```

Comment:
Your readme should describe the data structure and what each method does, not just be a modification on the class readme.